angular/core/services/chargify_service.coffee

```diff
@@ -0,0 +1,15 @@
+angular.module('loomioApp').factory 'ChargifyService', (CurrentUser) ->
+  new class ChargifyService
+
+    encodedParams: (group) ->
+      params =
+        first_name:   CurrentUser.firstName()
+        last_name:    CurrentUser.lastName()
+        email:        CurrentUser.email
+        organization: group.name
+        key:          group.key
```

Comment:
NB: We'll want to change the callback url from 'http://loomio.org/g/{reference}' to 'http://loomio.org/g/{key}' on the public chargify page in prod; I've already done this for the test chargify account.